### DISCUSSION TITLE
Can't activate email auth

### DISCUSSION THREAD

**Discussion by RomanMazyrin** [outline/outline#4066] [Self Hosting] [answered]
Hi there!

I tried to find information about my problem among issues and discussions, but i still have no clean answer. So here is my question:

I deployed outline on my own node on digital ocean, and it's totally okay. I deployed it on docker swarm with redis, postgres, https-portal and all the other necessary stuff. I ve set full set of settings in .env file by instruction, including SMTP server params and slack id/secret key. I am the only user in account right now and i authenticated myself first time via slack auth.

But, i still can not activate email auth, although it looks pretty right way. My checkbox in settings "Allow email authentication" is set to active, my SMTP settings is right, my teammates can receive invite links, but when they enter main login page, there is still only slack auth button.

The most interesting part is when i tested outline deployment on my localhost there was no any problems with it. I added my second email into account in "settings -> team" section, received invite email and after following the link in the incognito chrome mode i saw email auth form and successfully logged in.

Does anybody know what is going on and how to fix it?

**Comment by tommoor** [comment, marked as answer]
~Is email login enabled in Settings -> Security?~

Edit: `DEPLOYMENT` should not be included in environment variables.

**Reply by RomanMazyrin** [reply]
Yes

**Reply by tommoor** [reply]
Double check in the database that your `teams` table only has one entry

**Reply by RomanMazyrin** [reply]
Done, i checked, it is only one entry there.

**Reply by tommoor** [reply]
What is the output of `curl -X POST https://yourdomain.com/api/auth.config`

**Reply by RomanMazyrin** [reply]
{"data":{"providers":[{"id":"slack","name":"Slack","authUrl":"/auth/slack"}]},"status":200,"ok":true}

**Reply by tommoor** [reply]
You must have set the `DEPLOYMENT` env variable then, it should not be set for self-hosted

**Reply by RomanMazyrin** [reply]
Okay, but what options does this variable may have? Currently it is "hosted"

**Reply by tommoor** [reply]
It should not be set.

**Reply by RomanMazyrin** [reply]
Jesus, man, you are a real wizard! Thanks a lot, really appreciate it!
